scripts/startResourceServer.js

```diff
@@ -19,9 +19,20 @@ function startResourceServer() {
     return;
   }
 
-  const command = `mvn -Dokta.oauth2.issuer=${process.env.ISSUER} -f samples-java-spring-mvc/resource-server/pom.xml`;
+  if (process.env.CLIENT_ID === undefined) {
+    console.error('[ERROR] Please set the CLIENT_ID Environment variable');
+    return;
+  }
 
-  exec(command, {maxBuffer: 1024 * 2000}, (err, stdout) => {
+  const command = `node ${dir}/resource-server/server.js`;
+  const options = {
+    env: {
+      ISSUER: process.env.ISSUER,
+      SPA_CLIENT_ID: process.env.CLIENT_ID,
+      PATH: process.env.PATH
+    }
+  };
+  exec(command, options, (err, stdout) => {
```

Comment:
Is the missing maxBuffer significant?  Do we know why that was there?  (Just a Java thing?)

Reply:
I am assuming it was a Java thing.